app.ts

```diff
@@ -0,0 +1,285 @@
+    if (heading !== null) {
+        heading.innerHTML += headings;
+    }
+}
+
+// Table Content
+function createTableContent(contentData: string) {
+    const content: HTMLElement | null = document.getElementById("content");
+
+    let table = `<div id="row-${contentData[0]}" class="rows"></div>`;
+    if (content !== null) {
+        content.innerHTML += table;
+    }
+
+    let rows: HTMLElement | null = document.getElementById("row-" + contentData[0]);
+    for (let x = 0; x < contentData.length; x++) {
+        let rowCols = `<div class="row_cols">${contentData[x]}</div>`;
+        if (rows !== null) {
+            rows.innerHTML += rowCols;
+        }
+    }
+}
+
+//// Fetch Requests
+// Response Error Handling
+function handleErrors(response: Response) {
+    if (!response.ok) {
+        throw Error(response.statusText);
+    }
+    return response;
+}
+
+// Heading Row (Getting the columns data)
+function getHeadings() {
+    return fetch("http://localhost:2050/columns", {
+        method: "GET",
+        headers: { "Content-Type": "application/json" },
+    })
+        .then(handleErrors)
+        .then((response) => response.text())
+        .then((data) => {
+            let headingData = JSON.parse(data);
+            for (let i = 0; i < headingData.length; i++) {
+                createHeadingRow(headingData[i]);
+            }
+            getParameters(0);
+        })
+        .catch((error: Error) => {
+            console.log(error);
+        });
+}
+
+// Table Content (Getting the table's data)
+function getTable() {
+    const content: HTMLElement | null = document.getElementById("content");
+    let toParameter = getParameters(fromParameter);
+    const pageStats: HTMLElement | null = document.getElementById("pageStats");
+
+    // Clears Table
+    if (content !== null) {
+        content.innerHTML = "";
+    }
+
+    // Displays The Current Results Being Shown
+    let currentStats = `Showing results from ${fromParameter} to ${toParameter} out of ${recordCount} results.`;
+    if (pageStats !== null) {
+        pageStats.innerHTML = currentStats;
+    }
+
+    return fetch(`http://localhost:2050/records?from=${fromParameter}&to=${toParameter}`, {
+        method: "GET",
+        headers: { "Content-Type": "application/json" },
+    })
+        .then(handleErrors)
+        .then((res: Response) => res.text())
+        .then((data) => {
+            let contentData = JSON.parse(data);
+            for (let i = 0; i < contentData.length; i++) {
+                createTableContent(contentData[i]);
+            }
+        })
+        .catch((error: Error) => {
+            console.log(error);
+        });
+}
+
+// Gets Total Of All Records
+function getRecordCount() {
+    return fetch("http://localhost:2050/recordCount", {
+        method: "GET",
+        headers: { "Content-Type": "application/json" },
+    })
+        .then(handleErrors)
+        .then((res: Response) => res.text())
+        .then((data) => {
```

Comment:
Please remove the `()` or specify the type, as the compiler currently thinks the type is `any`